garak/generators/huggingface.py

```diff
@@ -168,19 +169,15 @@ class OptimumPipeline(Pipeline, HFCompatible):
     generator_family_name = "NVIDIA Optimum Hugging Face 🤗 pipeline"
     supports_multiple_generations = True
     doc_uri = "https://huggingface.co/blog/optimum-nvidia"
+    extra_dependency_names = ["optimum-nvidia"]
 
     def _load_client(self):
+        self._load_deps()
         if hasattr(self, "generator") and self.generator is not None:
             return
 
-        try:
-            from optimum.nvidia.pipelines import pipeline
-            from transformers import set_seed
-        except Exception as e:
-            logging.exception(e)
-            raise GarakException(
-                f"Missing required dependencies for {self.__class__.__name__}"
-            )
+        pipeline = self.optimum.nvidia.pipelines.pipeline
```

Comment:
Should this class be providing a `_load_deps()` override?

``` Python
    def _load_deps(self):
        return super()._load_deps(["optimum.nvidia"])
```

and accessing:

```suggestion
        pipeline = self.optimum_nvidia.pipelines.pipeline
```

Reply:
pruned optimum - it's EOL and currently incompatible due to dependencies